src/bridge/IrcBridge.ts

```diff
@@ -1111,4 +1111,13 @@ export class IrcBridge {
         }
         logCb(`Quit ${offlineCount}/${users.length}`);
     }
+
+    public async atBridgedRoomLimit() {
+        const limit = this.config.ircService.provisioning.roomLimit;
```

Comment:
`provisioning` is not required.
```suggestion
        const limit = this.config.ircService.provisioning?.roomLimit;
```